docs/workflows/workflow-triggers.md

```diff
@@ -0,0 +1,13 @@
+# Workflow Triggers
+
+For a workflow to run, it needs to be associated with a rule (see [rule configuration](../data-cookbooks/setup.md#rules). The rule configuration determines how and when a workflow execution is triggered. Rules can be triggered one time, on a schedule, or by new data written to a kinesis stream.
+
+There are three lambda functions in the API package responsible for scheduling and starting workflows: `SF scheduler`, `kinesis consumer`, `and SF starter`. Each Cumulus instance comes with a Start SF [SQS queue](https://aws.amazon.com/sqs/).
```

Comment:
Would it make sense to link to these files in the api package? On one hand, it would be easier than navigating through the repository for some users, but, on the other hand, I don't think the underlying mechanics of these lambda functions really comes into question here.